ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -302,24 +319,30 @@ runDataDiffusion tracers
                Left sd -> do
                    a <- Socket.getSocketName sd
                    case a of
-                        (Socket.SockAddrUnix path) ->
+                        (Socket.SockAddrUnix path) -> do
+                          traceWith dtDiffusionInitializationTracer $ DiffusionInitiatilizationCreatingLocalServerSocketForUnixPath path
```

Comment:
```suggestion
                          traceWith dtDiffusionInitializationTracer $ CreatingLocalServerSocket a
```
By including `a :: SockAddr` the message we don't need to have `ForUnixPath` prefix.